githubcontent/main.py

```diff
@@ -0,0 +1,132 @@
+from bs4 import BeautifulSoup
+import json
+import re
+import requests
+import uuid
+
+def get_github_urls(base_url: str, chapters: list) -> list:
+    """
+    get a list of urls 
+    """
+    urls = []
+    for chapter in chapters:
+        urls.append(base_url.format(chapter))
+        
+    print(urls[-5:])
+    return urls
+
+def parse_aantonop_books(urls):
+
+    documents = []
+    for url in urls:
+        is_bitcoin_url = re.search('bitcoinbook', url)
+        data = requests.get(url).text
+        soup = BeautifulSoup(data,'html.parser')
+        document = {}
+        title = soup.find('h2', dir='auto').get_text()
+        body = soup.find('div',id = 'readme').get_text()
+        body_type = "raw"
```

Comment:
They aren't raw though. They are asciidoc. We are throwing away the titles and headers but not persisting them.